Car-slider.html

```diff
@@ -0,0 +1,210 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <style>
+      * {
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+        outline: none;
+      }
+
+      .card {
+        display: inline-block;
+        background-color:#F6F6F8;
+        border-radius: 5px;
+        border: 1px solid #CCCCCC;
+        margin:3px;
+      }
+
+      .card-carousel {
+        width: 62%;
+        position: absolute;
+        top: 50%;
+        left: 50%;
+        transform: translate(-50%, -50%);
+        border-radius: 5px;
+      }
+
+      .card__information {
+        padding: 16px 12px 20px;
+      }
+
+      .card__name {
+        font-size: 16px;
+        color: #484848;
+        font-weight: 400;
+      }
+
+      .car__price {
+        font-size: 16px;
+        margin-top: 8px;
+        font-weight: 700;
+        color: #484848;
+      }
+
+      .card__avg-price {
+        font-size: 13px;
+        margin-top: 8px;
+        color: #6f6f6f;
+      }
+
+      .card__city-price-link {
+        color: #0288d1;
+        text-decoration: none;
+        margin-top: 8px;
+      }
+
+      .card__price-text {
+        font-size: 13px;
+        font-weight: 400;
+        color: #aaa;
+        padding-top: 2px;
+        padding-left: 3px;
+      }
+
+      .card-carousel__previous-slider,
+      .card-carousel__next-slider {
+        height: 32px;
+        width: 32px;
+        background-color: #d7d7d7;
+        position: absolute;
+        z-index: 1;
+        top: 50%;
+        transform:translateY(-50%);
+        border-radius: 50%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+
+      }
+
+      .card-carousel__next-slider {
```

Comment:
I think this is a modifier property so can we use a modifier name instead of the element?